apps/src/code-studio/pd/foorm/editor/foormEditorRedux.js

```diff
@@ -1,17 +1,17 @@
-const SET_FORM_QUESTIONS = 'foormEditor/SET_FORM_QUESTIONS';
-const SET_HAS_ERROR = 'foormEditor/SET_HAS_ERROR';
+const SET_QUESTIONS = 'foormEditor/SET_QUESTIONS';
+const SET_HAS_JSON_ERROR = 'foormEditor/SET_HAS_JSON_ERROR';
 const SET_FORM_DATA = 'foormEditor/SET_FORM_DATA';
-const RESET_AVAILABLE_FORMS = 'foormEditor/RESET_AVAILABLE_FORMS';
-const ADD_AVAILABLE_FORM = 'foormEditor/ADD_AVAILABLE_FORMS';
+const RESET_AVAILABLE_OPTIONS = 'foormEditor/RESET_AVAILABLE_OPTIONS';
```

Comment:
the meaning of "options" here needs a comment to explain that it's the actual editable object (forms, library questions). i also think "entities" might be a clearer name for those objects